packages/table/src/headers/columnHeaderCell.tsx

```diff
@@ -131,18 +136,19 @@ export class ColumnHeaderCell extends React.Component<IColumnHeaderCellProps, IC
     };
 
     public render() {
-        const { className, isActive, isColumnSelected, loading, resizeHandle, style } = this.props;
+        const { loading } = this.props;
```

Comment:
The props line got too long when I added my thing, plus elsewhere the Blueprint codebase, we don't deconstruct unless we need the same `this.props.X` variable in more than one place in the same scope. `loading` is the only prop here that's used in more than one place, so I left that and inlined everything else.